integration_tests/e2e/updateFunctionalityFull.cy.ts

```diff
@@ -200,7 +200,7 @@ context('Update functionality - Full flow', () => {
       )
 
       // When
-      particularJobInterestsPage.textField('OFFICE').type('A valid value')
+      particularJobInterestsPage.textField('OFFICE').type('Office administrator')
```

Comment:
Where possible I've not changed the cypress tests and what they do; and instead opted to change the data in the stubs (see below) to match the tests.
I felt it was better to change the data to match the existing tests, rather than change the tests to match the data

The only exception was this line that used a value of `A valid value` that made no sense at all! 🤣